packages/fastauth-metamask/.gitignore

```diff
@@ -0,0 +1,3 @@
+
```

Comment:
Is this file needed?

In this project, any dependencies need to be installed from the root package.json.